lib-app-bitcoin/context.c

```diff
@@ -15,24 +15,24 @@
 *  limitations under the License.
 ********************************************************************************/
 
-#include "btchip_internal.h"
-#include "btchip_apdu_constants.h"
+#include "context.h"
+#include "filesystem.h"
 
-#define MAX_LENGTH 248
+context_t context_D;
```

Comment:
Any reason for this naming?

Reply:
Nop' just old renaming of btchip_context_D to context_D, but I'll change it for sure